src/features/comment/CommentMarkdown.tsx

```diff
@@ -6,19 +7,27 @@ interface CommentMarkdownProps {
 }
 
 export default function CommentMarkdown({ children }: CommentMarkdownProps) {
+  const { renderCommentImages } = useAppSelector(
+    (state) => state.settings.general.comments,
+  );
+
   return (
     <Markdown
```

Comment:
Can we make this Markdown a StyledMarkdown and add css to target `img` inside with a max-height? Perhaps 200px will do.